omnibus/config/software/datadog-windows-filter-driver.rb

```diff
@@ -0,0 +1,21 @@
+name "datadog-windows-filter-driver"
+# at this moment,builds are stored by branch name.  Will need to correct at some point
+
+
+default_version "master"
+#
+# this should only ever be included by a windows build.
+if ohai["platform"] == "windows"
+    driverpath = ENV['WINDOWS_DDFILTER_DRIVER']
```

Comment:
no, because the path will change when we get a MS-certified driver.  This allows to switch between a stable certified driver and an 'unstable' one during development.